### PR TITLE
Windows fixes

### PR DESCRIPTION
**Pull request by aaron-boxer**
This PR contains a few small fixes to allow setup to run on Windows without erroring out.

### CODE REVIEW

**Comment by msaroufim** [comment]
Assuming CI passes this should be fine to merge, just triggered it now

**Comment by DiweiSun** [comment]
Regarding to the os.remove part, I suppose the temp file cannot be deleted since the file descriptor is not closed yet. Can we try with the following:

```
  +  fd, stdout_fpath = tempfile.mkstemp()

    try:
        output_buffer = io.FileIO(stdout_fpath, mode="w")
    except Exception as e:
        return (False, e, io.FileIO(stdout_fpath, mode="r").read().decode())
    finally:
        output_buffer.close()
        del output_buffer
        os.close(fd)
        os.remove(stdout_fpath)
```

**Comment by aaron-boxer** [comment]
> Regarding to the os.remove part in **init**.py, I suppose the temp file cannot be deleted since the file descriptor is not closed yet. Can we try with the following:
> 
> ```
>    fd, stdout_fpath = tempfile.mkstemp()
> 
>     try:
>         output_buffer = io.FileIO(stdout_fpath, mode="w")
>     except Exception as e:
>         return (False, e, io.FileIO(stdout_fpath, mode="r").read().decode())
>     finally:
>         output_buffer.close()
>         del output_buffer
>         os.close(fd)
>         os.remove(stdout_fpath)
> ```

Thanks, that worked - this is a much better fix.

File: torchbenchmark/__init__.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 import io
 import os
 import pathlib
+import platform
```
Comment (xuzhao9) [inline torchbenchmark/__init__.py:8]: Is this package being used in the code? Can we remove redundant dependency?

Comment (aaron-boxer) [reply torchbenchmark/__init__.py]: Good catch - I have removed it.

**Review by xuzhao9 — APPROVED** [review]
LGTM, thanks for fixing it on Windows!

**Pull request closed without merging**